extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -730,6 +773,35 @@ export class AmpA4A extends AMP.BaseElement {
     });
   }
 
+  renderViaSafeFrame_() {
+    user().assert(this.creativeBody_,
+        'creativeBody missing in renderViaSafeFrame_?');
+    this.lifecycleReporter_.sendPing('renderCrossDomainStart');
+    utf8Decode(this.creativeBody_).then(creative => {
+      /** @const {!Element} */
+      const iframe = this.element.ownerDocument.createElement('iframe');
+      iframe.setAttribute('height', this.element.getAttribute('height'));
```

Comment:
Just wondering: Is `allowfullscreen` something we typically put onto ad iframes?